web_widget_model_disable_nbrecords_computation/static/description/index.html

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE html
+    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Field omain widget disable nbRecords computation</title>
```

Comment:
Typo. And Line 19 same typo
```suggestion
    <title>Field domain widget disable nbRecords computation</title>
```